Public/Sdk/Public/Prelude/Prelude.OfficeBackCompatHacks.dsc

```diff
@@ -151,6 +151,8 @@ namespace Transformer {
         allowPreservedOutputs?: boolean;
         /** Obsolete: You must use import {Transformer} from "Sdk.Transformers" */
         passThroughEnvironmentVariables?: string[];
+        /** Obsolete: You must use import {Transformer} from "Sdk.Transformers" */
+        preserveOutputWhitelist?: (File | Directory)[];

```

Comment:
>preserveOutputWhitelist [](start = 8, length = 23)

Please do NOT add this one here...
TO use new API's office needs to modify the specgen, so they should at that time, add the import to the file.

This file is by now just here for the legacy branches which will not use that API.

Reply:
gotcha! removed.

---
In reply to: [287433544](https://github.com/microsoft/BuildXL/pull/369#discussion_r287433544) [](ancestors = 287433544)